internal/serialization/object_data_test.go

```diff
@@ -379,21 +366,22 @@ func TestObjectDataInput_ReadFloat64Array(t *testing.T) {
 	}
 }
 
-func TestObjectDataInput_ReadUTFArray(t *testing.T) {
+func TestObjectDataInput_ReadStringArray(t *testing.T) {
 	var array = []string{"aAüÜiİıIöÖşŞçÇ", "akdha", "üğpoıuişlk", "üğpreÜaişfçxaaöc"}
 	o := NewObjectDataOutput(0, nil, false)
 	o.WriteStringArray(array)
 	i := NewObjectDataInput(o.buffer, 0, nil, false)
-	retArray := i.ReadUTFArray()
+	retArray := i.ReadStringArray()
 
 	if !reflect.DeepEqual(array, retArray) {
 		t.Error("There is a problem in WriteStringArray() or ReadStringArray()!")
 	}
 }
 
+/*
```

Comment:
nit: `t.SkipNow()` should be used to skip tests. If this test is no longer valid, it's better to remove it completely

Reply:
I must have forgotten to uncomment that test.

Reply:
Removed at: https://github.com/hazelcast/hazelcast-go-client/pull/555/commits/f8335e3a66165c8383ce93ed08fc6127f3af1375